app/services/file_identifier_service.rb

```diff
@@ -39,7 +39,7 @@ def extract_file_types(output, filepath)
       next unless file['filename'] == filepath
 
       file['matches'].each do |match|
-        next unless match['ns'] == 'pronom'
+        next unless match['ns'] == 'pronom' || match['id'] == 'pronom'
```

Comment:
Can you say more about this change?

Reply:
When I was testing against the version of siegfried installed in the docker containers, it was using the `id` field instead of the `ns` field. This covers both cases.

Reply:
OK, I don't see a problem with that. What would you think about adding a code comment here?

Reply:
Good idea. Done.